ansible/roles/debops.prosody/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+---
+
+
+- name: Install required packages
+  package:
+    name: '{{ item }}'
+    state: 'present'
+  with_flattened:
+    - '{{ prosody__base_packages }}'
+
+- name: copy config
```

Comment:
I would rephrase it as 'Generate Prosody configuration' to be more descriptive about what the task does during the playbook execution.